### PR TITLE
imageio_libraw: minor error reporting improvements

### PR DESCRIPTION
**Pull request by kmilos**
...and also:

- backport https://github.com/LibRaw/LibRaw-cmake/commit/b82a1b0101b1e7264eb3113f1e6c1ba2372ebb7f (force enable OpenMP on MinGW)
- support building against stable LibRaw 0.20.x just in case

### CODE REVIEW

**Comment by kmilos** [comment]
@cytrinox Please review - I will also look into removing libraw_dcraw_process() next, it really bothers me :/

**Review by TurboGit — APPROVED** [review]
Looks good to me, I'll let @cytrinox comment before merging. Thanks.

**Comment by cytrinox** [comment]
Approved. If we can remove libraw_dcraw_process() in some way that whould be great.

**Comment by TurboGit** [comment]
Tested and working on my side so merging. Thanks!

**Comment by kmilos** [comment]
Btw, I have also found, at least for the M50, that the WB coeffs order is reverted, and I had to revert it back to the "normal" order...

**Comment by cytrinox** [comment]
@matt-maguire Can you confirm this? I can check it the next days, too.

**Comment by kmilos** [comment]
> If we can remove libraw_dcraw_process() in some way that whould be great

@cytrinox Ah, it looks like (only?) `idata.filters` changes when you do the processing step... So instead of that, perhaps we just do our own filters crop adjusting like [`imageio_rawspeed` does](https://github.com/darktable-org/darktable/blob/8367379638c820b56e0e233aeaec29709c6224e2/src/common/imageio_rawspeed.cc#L301)? I'll give that a go if someone doesn't beat me to it ;)

**Comment by kmilos** [comment]
@cytrinox Got one step further, it's not about cropping actually - unprocessed CR3 `idata.filters` seems to be of [4BAYER type (e.g. 0xb4b4b4b4)](https://github.com/darktable-org/darktable/blob/229d8496b4423c4f0224666d193a11fb88db79d4/src/common/imageio.h#L30-L37) that needs to be converted to a real 3 channel one like 0x94949494 (and what processing does for you). Then it works for me w/o calling `libraw_dcraw_process()`!

**Pull request merged**
